src/conn/conn_compact.c

```diff
@@ -115,6 +201,12 @@ __background_compact_should_run(WT_SESSION_IMPL *session, const char *uri, int64
 
     conn = S2C(session);
 
+    /* Check if the file is excluded. */
+    if (__background_compact_find_exclude_entry(session, uri)) {
```

Comment:
```__background_compact_find_exclude_entry``` could be renamed, eg: ```__background_compact_exclude```

Reply:
Renamed in cc52c025926072ed826202d782fc9641f879cd2a